newton/_src/utils/import_mjcf.py

```
@@ -484,6 +499,8 @@ def parse_body(
         joint_armature = []
         joint_name = []
         joint_pos = []
+        joint_custom_attributes: dict[str, Any] = {}
```

Ruff (0.14.2)

Undefined name `Any`

(F821)